crates/pipeline-types/src/config.rs

```diff
@@ -83,6 +72,18 @@ pub struct RuntimeConfig {
     #[serde(default = "default_workers")]
     pub workers: u16,
 
+    /// Should persistent storage be enabled for this pipeline?
+    ///
+    /// - If `true`, the pipeline state is stored in the specified location,
+    ///   is persisted across restarts, and can be checkpointed and recovered.
+    ///
+    /// - If `false`, the pipeline's state is kept in in-memory data-structures.
+    ///   This is useful if the pipeline is ephemeral and does not need to be recovered
+    ///   after a restart. The pipeline will most likely run faster since it does not
+    ///   need to read from, or write to disk
+    #[serde(default)]
+    pub storage: bool,
```

Comment:
False is the default right? Worth specifying for "true" that this is a preview/experimental feature..